dialect/sql/builder.go

```diff
@@ -616,6 +616,7 @@ func (d *DropIndexBuilder) Query() (string, []interface{}) {
 type InsertBuilder struct {
 	Builder
 	table     string
+	schema    string
```

Comment:
Let's move all the changes in this file to a separate PR, and add tests for it in the `builder_test.go` file.